app/backend/gwells/management/commands/export_databc.py

```diff
@@ -0,0 +1,304 @@
+                            access_key=get_env_variable(
+                                'S3_PUBLIC_ACCESS_KEY'),
+                            secret_key=get_env_variable(
+                                'S3_PUBLIC_SECRET_KEY'),
+                            secure=True)
+        for filename in files:
+            logger.info('uploading {}'.format(filename))
+            with open(filename, 'rb') as file_data:
+                file_stat = os.stat(filename)
+                target = 'api/v1/gis/{}'.format(filename)
+                bucket = get_env_variable('S3_WELL_EXPORT_BUCKET')
+                logger.debug(
+                    'uploading {} to {}/{}'.format(filename, bucket, target))
+                minioClient.put_object(bucket,
+                                       target,
+                                       file_data,
+                                       file_stat.st_size)
+
+    def write_record(self, f, record, fields, delim):
+        """ Write a single record to file."""
+        feature = {
+            'type': 'Feature',
+            'geometry': record[0],
+            'properties': {}
+        }
+        for col, value in enumerate(record):
+            if value is not None and fields[col] != 'geometry':
+                feature['properties'][fields[col]] = value
+        if delim:
+            f.write(',')
+        f.write('\r\n')
+        f.write(json.dumps(feature, cls=LazyEncoder))
+
+    def generate_geojson_chunks(self, sql, target, max_key, chunk_size):
+        """Generate GeoJSON in chunks small enough that we don't run out of memory,
+        or keep connections to the DB open for too long."""
+
+        # Keep track of the start time (purely for logging).
+        start = datetime.now()
+        # Open JSON file to write to.
+        with open(target, 'w') as f:
+            # Write the header - it's always the same.
+            self.write_geojson_header(f)
+            logger.info('{}: fetching spatial data from database...'.format(target))
+            current_index = 0
+            total_row_count = 0
+            more_chunks = True
+            # Loop until there are no more records.
+            while more_chunks:
+                with connection.cursor() as cursor:
+                    more_chunks = False
+                    logger.info('{target}: fetching {max_key} {current_index}...{next_index} '
+                                '({timestamp})'.format(target=target,
+                                                       max_key=max_key,
+                                                       current_index=current_index,
+                                                       next_index=current_index+chunk_size,
+                                                       timestamp=datetime.now() - start))
+                    # Get the next chunk from database.
+                    cursor.execute(sql.format(current_index, current_index+chunk_size))
```

Comment:
Could you change this to
`cursor.execute(sql, [current_index, current_index + chunk_size])`

The query will have to be changed to support that (using %s for placeholders).